backend/src/middleware/hashtags/hashtagsMiddleware.spec.js

```diff
@@ -36,7 +36,7 @@ beforeAll(() => {
     context: () => {
       return {
         user: authenticatedUser,
-        neode: instance,
+        neode,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T09:15:44Z" title="Friday, December 6th 2019, 10:15:44 am +01:00">Dec 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -36,7 +36,7 @@ beforeAll(() => {
     context: () => {
       return {
         user: authenticatedUser,
-        neode: instance,
+        neode: neode,
```


```suggestion
        neode,
```
can't we use ES6 syntax here?